community/ndp/transport-socket/src/main/java/org/neo4j/ndp/transport/socket/SocketProtocolV1.java

```diff
@@ -70,7 +71,7 @@ public SocketProtocolV1( final Log log, Session session )
         this.session = session;
         this.output = new ChunkedOutput();
         this.input = new ChunkedInput();
-        this.packer = new PackStreamMessageFormatV1.Writer( output, output.messageBoundaryHook() );
+        this.packer = new PackStreamMessageFormatV1.Writer( new PackStream.Packer( output ), output.messageBoundaryHook() );
```

Comment:
This is the reason I would like to keep the original constructor. This `Writer( Packer( ChunkedOutput ) )` is really too complicated to remember. For me, the `Packer` constructor is more designed for testing purpose.